src/components/TodoComponents/TodoList.js

```diff
@@ -6,7 +6,7 @@ import './Todo.css';
 
 const TodoList = props => {
     return (
-        <div>
+        <div className={'todoList'}>
```

Comment:
Why use braces here to evaluate the string? You can use just the string just fine:
```javascript
<div className='todoList'>
```
I'd also recommend keeping CSS to be kebab-cased over camelCased (so `todo-list` over `todoList` for the CSS class here).